history-of-quakers/modernized/09-chapter-7.adoc

```diff
@@ -410,15 +431,17 @@ It never rejoices but through sufferings; for with the world`'s joy it is murder
 I found it alone, being forsaken.
 I have fellowship therein with them who lived in dens, and desolate places of the earth,
 who through death obtained this resurrection, and eternal holy life.^
-footnote:[N. B. There is a passage in the book called, The complete History of England,
-vol.
-iii. page 201, which says that J. Nayler died with no fruits,
+footnote:[N. B. There is a passage in the book called, [.book-title]#The Complete History of England,
+Vol.
+iii#. page 201, which says that J. Nayler died with no fruits,
 nor so much as signs of repentance.
 How the author came by such information, we cannot tell,
 but that it is a manifest mistake we doubt not but
 the impartial reader is by this time abundantly convinced.]
+____
 
-JAMES NAYLER.
+[.signed-section-signature]
```

Comment:
Not sure if I should have used [sss] for his name, but I didn't embed it because it was a verbal quote.

Reply:
yup, makes sense. since we're going to switch to an embedded testimony, you can pull the signature up into the embed as well.